server/models/usermodel.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let UserSchema = new Schema({
+    first_name: {
+        type: String,
+        required: true
+    },
+    last_name: {
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    },
+    password: {
+        type: String,
+        required: true
+    },
+});
+
+UserSchema
+.virtual('name')
+.get(() => {
+    let fullname = '';
+    if (this.first_name && this.last_name) {
+        fullname = this.first_name + ' ' + this.last_name;
+    }
+    if (!this.first_name || !this.last_name) {
+        fullname = '';
+    }
```

Comment:
For this - I think we would want the name to be populated as the first_name if it exists and the last name does not exist

Reply:
Okay I will do it

Reply:
I made the change. Please review